client/data/README.md

```diff
@@ -0,0 +1,33 @@
+# Data
+
+This directory contains all services related to data fetching. Folders within this directory refer to domains of the application that are exposed through the services.
```

Comment:
The "service" term here sounds a bit contextual to Angular.js. Should we rephrase this a little bit so it could match the react context? One way to do this is to make it less abstract, like:

```
This directory contains hooks and high-order components related to data fetching. Folders within this directory refer to certain areas of the data that powers Calypso as an application.
```